help/en/package/apps/TabbedPreferences.shtml

```diff
@@ -573,23 +573,60 @@
       <h2 id="WiThrottle">WiThrottle</h2>
 
       <div>
+        <a href="images/PrefsWiThrottle.png"><img src="images/PrefsWiThrottle.png" height=
+        "530" width="480" class="floatRight" alt="JMRI v5.12 WiThrottle Preferences Tab"></a>
+
         <dl>
           <dt>eStop</dt>
 
           <dd>
-            If selected, listens for a packet to be sent from the connected mobile device. If
-            WiThrottle Server does not receive any communication in the chosen time, the affected
-            JMRI throttle will idle its train.
+            <p>If selected: If the WiThrottle Server does not receive any communication in the chosen time,
+            the affected JMRI throttle will send an emergency stop command to its train.</p>
             <p>The chosen time is approximate; the response may be a few seconds earlier or
             later.</p>
           </dd>
 
+          <dt>Function Buttons</dt>
+
+          <dd>
+            If selected, the F2 key on WiThrottle clients will always be treated as a momentary function.
+            This setting overrides any setting from the roster and it cannot be overridden by
+            WiThrottle clients.
+          </dd>
+
+          <dt>Acquire address silently if already in use</dt>
+
+          <dd>
+            By default, the WiThrottle server will silently share access to a certain loco between all
+            WiThrottle clients and JMRI throttles. If this option is selected, it will instead send a
+            "Steal from other WiThrottle or JMRI throttle Required" message to the client and wait for
+            the client to confirm the steal if the loco is already in use. See the
+            <a href="#Throttle">Throttle</a> section for stealing / sharing interaction with hardware
+            throttles from your DCC system.
```

Comment:
I'm not sure I understand this text. On one hand, it seems that you mean that the WiThrottle will steal the throttle from other JMRI clients, including other WiThrottles.

On the other hand, the last sentence tells about stealing from hardware throttles.